src/main/java/model/Player.java

```diff
@@ -0,0 +1,28 @@
+package main.java.model;
+
+public class Player {
+	private String name;
+	private int cell;

```

Comment:
should cell be a part of the Player class?

Reply:
to keep track of where a current player is at